core/app/c/[communitySlug]/forms/[formSlug]/edit/EditFormTitleButton.tsx

```diff
@@ -0,0 +1,106 @@
+"use client";
+
+import { useState } from "react";
+import { zodResolver } from "@hookform/resolvers/zod";
+import { useForm } from "react-hook-form";
+import { z } from "zod";
+
+import { Button } from "ui/button";
+import { Dialog, DialogContent, DialogFooter, DialogTitle, DialogTrigger } from "ui/dialog";
+import {
+	Form,
+	FormControl,
+	FormDescription,
+	FormField,
+	FormItem,
+	FormLabel,
+	FormMessage,
+} from "ui/form";
+import { Input } from "ui/input";
+import { toast } from "ui/use-toast";
+
+import { didSucceed, useServerAction } from "~/lib/serverActions";
+import { updateForm } from "./actions";
+
+const editFormTitleSchema = z.object({
+	name: z.string().min(1, "Form name is required"),
+});
+
+const EditFormTitleButton = ({ name, communityId }: { name: string; communityId: string }) => {
+	const [isOpen, setIsOpen] = useState(false);
+
+	const form = useForm<z.infer<typeof editFormTitleSchema>>({
+		resolver: zodResolver(editFormTitleSchema),
+		defaultValues: {
+			name,
+		},
+	});
+
+	const runUpdateFormTitle = useServerAction(updateForm);
+	const onSubmit = (data: z.infer<typeof editFormTitleSchema>) => {
+		const result = runUpdateFormTitle({ communityId, name: data.name });
+		if (didSucceed(result)) {
+			toast({
+				title: "Success",
+				description: "Form name was successfully updated",
```

Comment:
<img width="636" alt="image" src="https://github.com/user-attachments/assets/b1798ec5-0488-4a75-94ec-11004ac1d838">
slightly different wording in the designs

Reply:
This should also be centered, green, and have the check, like the save toast https://github.com/pubpub/v7/blob/b94ac8bbbdd4533305cc08e8affdf13ff35f9ac9/core/app/components/FormBuilder/FormBuilder.tsx#L178-L185

could you extract that config to its own toast component (or make it an option on the existing one) and then use it here so they have the same style? as a bonus you could also make it actually centered, since I seem to have missed that before!